configuration/jobs/seed-multibranch-pipelines/seed-multibranch-pipeline.groovy

```diff
@@ -21,23 +21,31 @@ def githubAccount = System.getenv("GITHUB_ACCOUNT")
 def githubOrg = System.getenv("GITHUB_ORG") ?: false
 //eg  https://api.github.com/users/springdo/repos or 
 
-def githubProjects = githubOrg ? new URL("${githubHost}/orgs/${githubAccount}/repos?per_page=100") : new URL("${githubHost}/users/${githubAccount}/repos?per_page=100")
+// BITBUCKET
+def bitbucketHost = System.getenv("BITBUCKET_HOST") ?: "https://bitbucket.org/repo"
+def bitbucketUser = System.getenv("BITBUCKET_USER")
+def bitbucketPassword = System.getenv("BITBUCKET_PASSWORD")
+def bitbucketProjectKey = System.getenv("BITBUCKET_PROJECT_KEY") ?: "rht-labs"
+def bitbucketProjectsApi = new URL("${bitbucketHost}/rest/api/1.0/projects/${bitbucketProjectKey}/repos?limit=100")
+def bitbucketAuth = (bitbucketUser+":"+bitbucketPassword).getBytes().encodeBase64().toString();
 
+def githubProjects = githubOrg ? new URL("${githubHost}/orgs/${githubAccount}/repos?per_page=100") : new URL("${githubHost}/users/${githubAccount}/repos?per_page=100")
 
 def createMultibranchPipelineJob(project, gitPath, jte) {
-    def buildNamespace = System.getenv("BUILD_NAMESPACE") ?: "labs-ci-cd"
+    def buildNamespace = System.getenv("BUILD_NAMESPACE") ?: "ocp-ci-cd"
     def buildGitAuthSecret = System.getenv("BUILD_GIT_AUTH_SECRET") ?: "git-auth"
     def jteProject = System.getenv("JTE_PROJECT") ?: "https://gitlab.apps.proj.example.com/rht-labs/pipeline-template-configuration.git"
     def pipelineConfigDir = System.getenv("JTE_PIPELINE_DIR") ?: "pipeline-configuration"
     def librariesDir = System.getenv("JTE_LIBRARIES_DIR") ?: "libraries"
+    def credentialsId = System.getenv("SSH_ACCESS_KEY") ?: "Access-Key"
 
     // Build Jenkins multibranch jobs
     multibranchPipelineJob(project) {
         branchSources {
             git {
                 id("${project}")
                 remote(gitPath)
-                credentialsId("${buildNamespace}-${buildGitAuthSecret}")
+                credentialsId("${credentialsId}")
```

Comment:
Not sure if we need this. I think, instead of defining a new SSH_ACCESS_KEY variable, you can create a secret with ssh key in it (secret type `kubernetes.io/ssh-auth`). It'd be the same as username/password secret. And you get to use `BUILD_GIT_AUTH_SECRET` and doesn't have to change `credentialsId`